### DISCUSSION TITLE
Why are you using pacdef?

### DISCUSSION THREAD

**Discussion by keilmillerjr** [steven-omaha/pacdef#49] [General]
NixOS seems to be making some news lately. I like the idea of defining packages in a way that I can refer back to. I could also reproduce the same system should I start again with a fresh install. I do not like that nix does not adhear to FHS, requires a ton of patching, and lots of configuration. Pacdef seems to meet the criteria of what I like, while keeping it simple. Thank you for this wonderful utility.

**Comment by steven-omaha** [comment]
Glad that you like it!

I wrote pacdef because I often found that when I added a tool on one computer, I quickly missed it on my others. Pacdef solved that. It's that simple. 

Additionally, I can now look up what packages I have installed on any machine quickly, and can install all packages I need for a specific use case with two simple commands if I need to do something on a computer that I wouldn't normally do there.

I tried Nix once, but couldn't get zfs to work. Since I use zfs for almost everything, Nix wasn't for me.
